tests/test_helpers.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
Perhaps call the file ``helpers.py`` instead, to avoid pytest thinking the file contains test cases.